Hints.py

```diff
@@ -142,7 +142,7 @@ def isRestrictedDungeonItem(dungeon, item):
     return False
 
 
-def add_hint(spoiler, world, groups, gossip_text, count, location=None, force_reachable=False):
+def add_hint(spoiler, world, groups, gossip_text, count, location=None, secondLocation=None, force_reachable=False):
```

Comment:
This would be cleaner if `location` could be either a single location or a list. Convert single locations to a list with one element and then loop through for each hinted location. This also has the benefit of supporting more than 2 locations for a hint if that gains traction.